### PR TITLE
Swap hhhh

### PR DESCRIPTION
**Pull request by EmperorRay**
hahahahaha

### CODE REVIEW

**Comment by SuboOx** [comment]
There are some redundant code.

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
 
 import java.util.Random;
-
+/////
```
Comment (SuboOx) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:7]: This line makes no sense.

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
 
 import java.util.Random;
-
```
Comment (SuboOx) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:7]: You should leave a blank line here.

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 1 above)
Comment (EmperorRay) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:7]: NO you are wrong!

**Review by SuboOx — APPROVED** [review]
Good job fixing the bug.

**Pull request merged**